world_boss/app/raid.py

```diff
@@ -241,3 +243,17 @@ def get_currencies(db: Session) -> List[CurrencyDictionary]:
         }
         result.append(currency)
     return result
+
+
+def get_jwt_auth_header() -> dict[str, str]:
+    encoded = jwt.encode(
+        {
+            "exp": (datetime.datetime.now() + datetime.timedelta(hours=1))
```

Comment:
1 hour lifetime is too long for HTTP header token.
Unlike blockchain Tx, JWT header is newly generated everytime just before request.